common/src/commonMain/kotlin/entity/DiscordMessage.kt

```diff
@@ -534,7 +542,10 @@ public data class DiscordAttachment(
     */
     val width: OptionalInt? = OptionalInt.Missing,
 
-    val ephemeral: OptionalBoolean = OptionalBoolean.Missing
+    val ephemeral: OptionalBoolean = OptionalBoolean.Missing,
+    @SerialName("duration_secs")
+    val durationSecs: Optional<Float> = Optional.Missing(),
```

Comment:
i think we should introduce a new `Duration` serializer for this, also `Double` is probably better